docs/deploying-airbyte/on-oci-vm.md

```diff
@@ -63,34 +101,47 @@ wget https://raw.githubusercontent.com/airbytehq/airbyte/master/{.env,docker-com
 
 which docker-compose
 
-docker-compose up -d
+sudo /usr/local/bin/docker-compose up -d
```

Comment:
Do we have to reference the binary location specifically? Does it not work otherwise?

Reply:
In Oracle Linux 7 we need to reference the binary location else it does not work.